swift/ql/lib/codeql/swift/elements/expr/Expr.qll

```diff
@@ -7,6 +7,12 @@ class Expr extends ExprBase {
 
   Expr getConversion() { result.convertsFrom(this) }
 
+  Expr getConversion(int n) {
+    n = 0 and result = this.getConversion()
+    or
+    result = this.getConversion(n - 1).getConversion()
+  }
+
```

Comment:
I suppose this predicate is unused now, but maybe it'll be a useful thing to have in the future. If so, we should maybe add a `Expr.getAConversion()` predicate as well in the future.